interactive

```diff
@@ -26,6 +26,12 @@ Useful arg examples:
                          (note this is not GPU memory)
 * --cpus-per-task=1      give me 1 cpu, there's 32 on most
                          nodes
+* --partition=...        there is a partition explicitly for
+                         interactive jobs (called Partition).
```

Comment:
The partition name should be 'Interactive', not 'Partition' in the welcome message.

Reply:
Duh...cheers man